test/test.pl

```diff
@@ -2163,6 +2163,7 @@ sub test_bam2fq
             $test++;
         }
     }
+    test_cmd($opts,out_map=>{'1.fq' => 'bam2fq/1.1.fq.expected', '2.fq' => 'bam2fq/1.2.fq.expected'},cmd=>"$$opts{bin}/samtools bam2fq -1 $$opts{path}/1.fq -2 $$opts{path}/2.fq $$opts{path}/dat/bam2fq.001.sam");
```

Comment:
No test with `-s`, so `bam2fq_mainloop_singletontrack()` is not exercised in the test suite.